test/e2e-tests.js

```diff
@@ -541,6 +541,20 @@ lab.experiment("core rendering-info", () => {
       );
     }
   );
+
+  it(
+    "correctly merges additionalRenderingInfo configured in targets and tools",
+    { plan: 1 },
+    async () => {
```

Comment:
with only this one test, this would not fail if `additionalRenderingInfo.sophieModules` from target config includes `sophie-q` but would overwrite  `additionalRenderingInfo.sophieModules` from tool endpoint config.

Please add some more tests to make sure everything is in the final `renderingInfo` and merged as intended.